ElectronClient/plugins/GotoAnything.jsx

```diff
@@ -208,7 +208,7 @@ class Dialog extends React.PureComponent {
 				searchQuery = this.makeSearchQuery(this.state.query);
 				results = await SearchEngine.instance().search(searchQuery);
 
-				resultsInBody = !!results.find(row => row.fields.includes('body'));
+				resultsInBody = !!results.find(row => row.fields ? row.fields.includes('body') : !row.title.includes(this.state.query));
```

Comment:
I think instead we should make the search engine return a consistent, so we should add the "fields" property to the results.

Reply:
ok, I will try

Reply:
@laurent22 I've pushed another commit, checking it out and giving some advice would be appreciated